### PR TITLE
Feature/cibuild

### PR DESCRIPTION
**Pull request by maribel4amido**
Update gradle to support flutter v1.7

Update CI workflows to create an appbundle and upload to  sharing or internal test track.

Builds are triggered for commits:
1. develop, feature/* - this is qa flavor uploaded to sharing only
2. rcX.Y.Z tag - this is production flavor uploaded to sharing only
3. release/* - this is production flavor uploaded to internal test track ready for open beta
4. vA.B.Z tag - this is production flavor uploaded to internal test track ready for open beta

### CODE REVIEW

File: lib/ui/home.dart [hunk 1]
```diff
@@ -160,7 +160,7 @@ class _HomeState extends State<Home> with WidgetsBindingObserver {
 
   Future<void> _retrieveDynamicLink(HomeViewmodel viewModel) async {
     final PendingDynamicLinkData data =
-        await FirebaseDynamicLinks.instance.retrieveDynamicLink();
+        await FirebaseDynamicLinks.instance.getInitialLink();
```
Comment (LeeMatthewHiggins) [inline lib/ui/home.dart:163]: looks good 👍

Comment (LeeMatthewHiggins) [reply lib/ui/home.dart:163]: @DavidFD005 you will need this change for app entry..

**Review by LeeMatthewHiggins — APPROVED** [review]
Looks good to me 👍

File: lib/ui/home.dart (hunk 1 above)
Comment (davidfranquet) [reply lib/ui/home.dart:163]: I've already changed this :) because I've update the firebase dynamic link lib

**Review by davidfranquet — APPROVED** [review]

**Pull request merged**